corehq/tests/app_test_db_dependencies.yml

```diff
@@ -5,6 +5,19 @@
 #
 # This means that replacing INSTALLED_APPS with the list of declared dependencies plus the app
 # should still allow `./manage.py test app` to pass.
+
+# apps listed in the 'default' section will be included for all other apps
+default:
+  - django.contrib.auth
+  - django.contrib.contenttypes
+  - django.contrib.sessions
+  - two_factor
+  - django_otp.plugins.otp_static
+  - django_otp.plugins.otp_totp
+  - corehq.couchapps
+  - corehq.apps.tzmigration
+  - couchforms
+  - corehq.apps.users
```

Comment:
Just to confirm, this makes all of these dependencies of any DB test, yeah? I have mixed feelings about this. I see how it does make updating this list easier, but it will also make many tests that have fewer dependencies much much slower. Can you think of any way to prevent that second thing from happening?